<launch>
  <node name="rgb_cam" pkg="ids_cam_driver" type="ids.py" output="screen" >
    <param name="width" value="640" />
    <param name="height" value="480" />

  </node>

  <node name="rgb_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/cam1/image_raw"/>
    <param name="autosize" value="true"/>
  </node>

  <!-- event camera driver -->
  <node name="dvxplorer_ros_driver" pkg="dvxplorer_ros_driver" type="dvxplorer_ros_driver" />
  <!--param name="imu_bias/ax" value="0.0001"/-->
  <!-- events visualization -->
  <node name="dvs_renderer" pkg="dvs_renderer" type="dvs_renderer">
    <!-- <param name="display_method" value="grayscale"/> -->
    <param name="display_method" value="red-blue"/>

    <remap from="events" to="/dvs/events" />
    <remap from="camera_info" to="/dvs/camera_info" />
  </node>
  <node name="event_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/dvs_rendering"/>
    <param name="autosize" value="true" />
  </node>
</launch>
